extensions.json

```
@@ -653,6 +653,11 @@
       "checkout": "v1.9.0",
       "location": "packages/client"
     },
+    {
+      "id": "svipas.control-snippets",
+      "repository": "https://github.com/svipas/vscode-control-snippets",
+      "version": "1.9.0"
```

The repo seems to have a dedicated Git tag for this release, so I think it's safer to publish from that instead of from the default branch.

```suggestion
      "version": "1.9.0",
      "checkout": "1.9.0"
```